requests/status_codes.py

```diff
@@ -89,3 +89,18 @@
         setattr(codes, title, code)
         if not title.startswith('\\'):
             setattr(codes, title.upper(), code)
+
+_success = {
+    200,
+    201,
+    202,
+    203,
+    204,
+    205,
+    206,
+    207,
+    208,
+    226,
+}
+
+setattr(codes, 'success', _success)
```

Comment:
I think this is over-engineering the solution. This also isn't the appropriate object to tack "success" onto when it's not a status code and the attribute you're setting is not an integer (like everything else on codes has been historically).

I'm okay having something like

```py
# requests/status_codes.py
success_set = {
    # ...
}
```

```py
# requests/models.py
from . import status_codes

class Response(..):
    @property
    def ok(self):
        return self.status_code in status_codes.success_set
```